### ISSUE TITLE
Is there anybody out there?

### ISSUE THREAD

**Issue by dfischercodethoughts** [mhjlam/BAM#1]
Found this from William SRD's youtube channel. Anyone working on this? Could probably port this into a phone game.

**Comment by mhjlam** [comment]
Nope, nobody is working on this as far as I know. This is just the interactive demo code as-is taken from some CD. It doesn't even compile on modern systems. I was once planning to make it compilable again but abandoned that project.
